week1/citibike.R

```diff
@@ -23,26 +23,47 @@ trips <- mutate(trips, gender = factor(gender, levels=c(0,1,2), labels = c("Unkn
 ########################################
 
 # count the number of trips (= rows in the data frame)
-
+nrow(trips)
 # find the earliest and latest birth years (see help for max and min to deal with NAs)
-
+max(trips$birth_year)
+trips$birth_year[trips$birth_year != "\\N"] %>% 
+ min()
```

Comment:
This is a totally valid approach, but there are cases in other datasets where it might not work since the NANs might be recorded in a more sensible way. Luckily, `max()` and `min()` have a flag that automatically gets rid of non-numeric data which might be useful in the future.